share/drom/skeletons/projects/ppx_rewriter/drom.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+drom-version = "0.1.0"
+
+[project]
+authors = [!{authors-for-toml}]
+copyright = "!{copyright}"
+edition = "!{edition}"
+github-organization = "!{github-organization}"
+license = "!{license-name}"
+min-edition = "!{min-edition}"
+mode = "binary"
+name = "!{name}"
+synopsis = "The !{name} project"
+version = "0.1.0"
+windows-ci = true
+
+# keys that you could also define:
+# sphinx-target = "...sphinx-target..."
+# archive = "...archive..."
+# dev-repo = "...dev-repo..."
+# bug-reports = "...bug-reports..."
+# doc-api = "...doc-api..."
+# doc-gen = "...doc-gen..."
+# homepage = "...homepage..."
+
+[project]
+description = """
+This is the description
+of the !{name} OCaml project
+"""
+
+[drom]
+skip = ""
+
+[dependencies]
+
+[tools.ocamlformat]
+for-test = true
+[tools.odoc]
+for-doc = true
+[tools.ppx_expect]
+for-test = true
+[tools.ppx_inline_test]
+for-test = true
+
+[project]
+pack-modules = true
+skip-dirs = []
+
+[[package]]
+skeleton = "ppx_rewriter"
+dir = "src/!{name}"
+kind = "library"
```

Comment:
This can be removed as it is already in `share/drom/skeletons/packages/ppx_rewriter/package.toml`.

Reply:
thanks, I had just copied the ppx_deriver skeleton files

Reply:
I guess it hasn't been updated when we added support for `package.toml` files. Could you fix it in this PR by just removing the redundant line.

Reply:
yes that's what I did

Reply:
I meant the one in `ppx_deriver`